contents.json

```diff
@@ -1254,8 +1254,11 @@
             "title": "Ghostery",
             "category-ids": ["browser"],
             "tags": ["swift"],
+            "homepage": "https://www.ghostery.com/mobile/",
             "description": "Ghostery Browser",
             "source": "https://github.com/ghostery/browser-ios",
+            "license": "Mozilla Public License 2.0",
```

Comment:
@ebal could you replace it by the following code according to the contribution guidelines `mpl-2.0`

> license: Please use the key name from https://developer.github.com/v3/licenses/ (use other if license info missing)